.vib/apache/goss/goss.yaml

```diff
@@ -0,0 +1,15 @@
+http:
+  https://apache:{{ .Vars.service.ports.https }}:
+    status: 200
+    allow-insecure: true
+file:
+  /etc/hosts:
+    exists: true
+    filetype: file
+    contains:
+    {{ range .Vars.hostAliases }}
+    {{ $ip := .ip }}
+    {{ range $host := .hostnames }}
+      - /{{ $ip }}.*{{$host}}/
+    {{ end }}
+    {{ end }}
```

Comment:
We are adding a section for `podSecurityContext` and `containerSecurityContext` now by convention. Please, add this to the test:

```
command:
  check-user-info:
    exec: id
    exit-status: 0
    stdout:
      - uid={{ .Vars.containerSecurityContext.runAsUser }}
      - /groups=.*{{ .Vars.podSecurityContext.fsGroup }}/
```